### ISSUE TITLE
Bootstrap error

### ISSUE THREAD

**Issue by ralpha** [rust-lang/rust-analyzer#13083]
Steps:
Opened up project, disabled `CodeLLDB` extension. Releaded VSCode -> Error. Reload again, no change.

Output:
```
INFO [8/22/2022, 1:16:53 PM]: PersistentState: { serverVersion: '0.3.1178' }
INFO [8/22/2022, 1:16:54 PM]: Using server binary at /home/user/.vscode/extensions/rust-lang.rust-analyzer-0.3.1178-linux-x64/server/rust-analyzer
ERROR [8/22/2022, 1:16:54 PM]: Bootstrap error [Error: Failed to execute /home/user/.vscode/extensions/rust-lang.rust-analyzer-0.3.1178-linux-x64/server/rust-analyzer --version
	at IS (/home/user/.vscode/extensions/rust-lang.rust-analyzer-0.3.1178-linux-x64/out/main.js:85:2328)
	at processTicksAndRejections (node:internal/process/task_queues:96:5)
	at async kS (/home/user/.vscode/extensions/rust-lang.rust-analyzer-0.3.1178-linux-x64/out/main.js:84:1060)
	at async pv (/home/user/.vscode/extensions/rust-lang.rust-analyzer-0.3.1178-linux-x64/out/main.js:84:891)
	at async v._activate (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:8180)
	at async v._waitForDepsThenActivate (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:8122)
	at async v._initialize (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:7486)]
INFO [8/22/2022, 1:16:54 PM]: Extension version: 0.3.1178
```

**rust-analyzer version**: 0.3.1178

**rustc version**: 1.62.1

 
<details>
  <summary>relevant settings:</summary>

```js
{
  cargoRunner: null,
  runnableEnv: null,
  server: { path: null, extraEnv: null },
  trace: { server: 'off', extension: false },
  debug: {
    engine: 'auto',
    sourceFileMap: {
      '/rustc/<id>': '${env:USERPROFILE}/.rustup/toolchains/<toolchain-id>/lib/rustlib/src/rust'
    },
    openDebugPane: false,
    engineSettings: {}
  },
  restartServerOnConfigChange: false,
  typing: {
    continueCommentsOnNewline: true,
    autoClosingAngleBrackets: { enable: false }
  },
  assist: { expressionFillDefault: 'todo' },
  cachePriming: { enable: true, numThreads: 0 },
  cargo: {
    autoreload: true,
    buildScripts: { enable: true, overrideCommand: null, useRustcWrapper: true },
    features: [],
    noDefaultFeatures: false,
    noSysroot: false,
    target: null,
    unsetTest: [ 'core' ]
  },
  checkOnSave: {
    allTargets: true,
    command: 'clippy',
    enable: true,
    extraArgs: [ '--target-dir', 'target/rls/' ],
    features: null,
    noDefaultFeatures: null,
    overrideCommand: null,
    target: null
  },
  completion: {
    autoimport: { enable: true },
    autoself: { enable: true },
    callable: { snippets: 'fill_arguments' },
    postfix: { enable: true },
    privateEditable: { enable: false },
    snippets: {
      custom: {
        'Arc::new': {
          postfix: 'arc',
          body: 'Arc::new(${receiver})',
          requires: 'std::sync::Arc',
          description: 'Put the expression into an `Arc`',
          scope: 'expr'
        },
        'Rc::new': {
          postfix: 'rc',
          body: 'Rc::new(${receiver})',
          requires: 'std::rc::Rc',
          description: 'Put the expression into an `Rc`',
          scope: 'expr'
        },
        'Box::pin': {
          postfix: 'pinbox',
          body: 'Box::pin(${receiver})',
          requires: 'std::boxed::Box',
          description: 'Put the expression into a pinned `Box`',
          scope: 'expr'
        },
        Ok: {
          postfix: 'ok',
          body: 'Ok(${receiver})',
          description: 'Wrap the expression in a `Result::Ok`',
          scope: 'expr'
        },
        Err: {
          postfix: 'err',
          body: 'Err(${receiver})',
          description: 'Wrap the expression in a `Result::Err`',
          scope: 'expr'
        },
        Some: {
          postfix: 'some',
          body: 'Some(${receiver})',
          description: 'Wrap the expression in an `Option::Some`',
          scope: 'expr'
        }
      }
    }
  },
  diagnostics: {
    disabled: [],
    enable: true,
    experimental: { enable: false },
    remapPrefix: {},
    warningsAsHint: [],
    warningsAsInfo: []
  },
  files: { excludeDirs: [], watcher: 'client' },
  highlightRelated: {
    breakPoints: { enable: true },
    exitPoints: { enable: true },
    references: { enable: true },
    yieldPoints: { enable: true }
  },
  hover: {
    actions: {
      debug: { enable: true },
      enable: true,
      gotoTypeDef: { enable: true },
      implementations: { enable: true },
      references: { enable: false },
      run: { enable: true }
    },
    documentation: { enable: true, keywords: { enable: true } },
    links: { enable: true }
  },
  imports: {
    granularity: { enforce: false, group: 'crate' },
    group: { enable: true },
    merge: { glob: true },
    prefix: 'plain'
  },
  inlayHints: {
    bindingModeHints: { enable: false },
    chainingHints: { enable: false },
    closingBraceHints: { enable: false, minLines: 25 },
    closureReturnTypeHints: { enable: 'never' },
    lifetimeElisionHints: { enable: 'never', useParameterNames: false },
    maxLength: 25,
    parameterHints: { enable: false },
    reborrowHints: { enable: 'never' },
    renderColons: false,
    typeHints: {
      enable: false,
      hideClosureInitialization: false,
      hideNamedConstructor: false
    }
  },
  joinLines: {
    joinAssignments: true,
    joinElseIf: true,
    removeTrailingComma: true,
    unwrapTrivialBlock: true
  },
  lens: {
    debug: { enable: true },
    enable: true,
    forceCustomCommands: true,
    implementations: { enable: true },
    references: {
      adt: { enable: false },
      enumVariant: { enable: false },
      method: { enable: false },
      trait: { enable: false }
    },
    run: { enable: true }
  },
  linkedProjects: [],
  lru: { capacity: null },
  notifications: { cargoTomlNotFound: true },
  procMacro: {
    attributes: { enable: true },
    enable: false,
    ignored: {},
    server: null
  },
  runnables: { command: null, extraArgs: [ '--target-dir', 'target/rls/' ] },
  rustc: { source: null },
  rustfmt: {
    extraArgs: [],
    overrideCommand: null,
    rangeFormatting: { enable: false }
  },
  semanticHighlighting: { strings: { enable: true } },
  signatureInfo: { detail: 'full', documentation: { enable: true } },
  workspace: {
    symbol: { search: { kind: 'only_types', limit: 128, scope: 'workspace' } }
  }
}
```

</details>

**Comment by ralpha** [comment]
Update: Switching to pre-release version does not change/fix the issue.

```
INFO [8/22/2022, 1:32:20 PM]: PersistentState: { serverVersion: '0.3.1178' }
INFO [8/22/2022, 1:32:20 PM]: Using server binary at /home/user/.vscode/extensions/rust-lang.rust-analyzer-0.4.1177-linux-x64/server/rust-analyzer
ERROR [8/22/2022, 1:32:20 PM]: Bootstrap error [Error: Failed to execute /home/user/.vscode/extensions/rust-lang.rust-analyzer-0.4.1177-linux-x64/server/rust-analyzer --version
	at IS (/home/user/.vscode/extensions/rust-lang.rust-analyzer-0.4.1177-linux-x64/out/main.js:85:2328)
	at processTicksAndRejections (node:internal/process/task_queues:96:5)
	at async kS (/home/user/.vscode/extensions/rust-lang.rust-analyzer-0.4.1177-linux-x64/out/main.js:84:1060)
	at async pv (/home/user/.vscode/extensions/rust-lang.rust-analyzer-0.4.1177-linux-x64/out/main.js:84:891)
	at async v._activate (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:8180)
	at async v._waitForDepsThenActivate (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:8122)
	at async v._initialize (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:7486)]
```

So maybe it is related to settings? But did not change any settings recently.

**Comment by lnicola** [comment]
Duplicate of https://github.com/rust-lang/rust-analyzer/issues/13081.
